storage/innobase/row/row0import.cc

```diff
@@ -1335,12 +1335,15 @@ row_import::match_schema(
 	THD*		thd) UNIV_NOTHROW
 {
 	/* Do some simple checks. */
+	const unsigned relevant_flags = m_flags & ~DICT_TF_MASK_DATA_DIR;
+	const unsigned relevant_table_flags
+		= m_table->flags & ~DICT_TF_MASK_DATA_DIR;
 
-	if (m_flags != m_table->flags) {
+	if (relevant_flags != relevant_table_flags) {
```

Comment:
I am not sure - because the user has no means of actually fixing this warning short of hex editor. A warning that's unfixable would be error log spam